Dockerfile

```
@@ -102,8 +102,7 @@ RUN git clone https://github.com/aiidalab/aiidalab-optimade.git && cd aiidalab-o
 RUN pip install -e ./aiidalab-optimade
 
 # Install some useful packages that are not available on PyPi
-RUN conda install --yes -c conda-forge rdkit
-RUN conda install --yes -c openbabel openbabel
+RUN conda install --yes -c conda-forge rdkit openbabel
```

I think we should structure the `conda install` commands like [`apt install` commands](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#sort-multi-line-arguments):
```suggestion
RUN conda install --yes -c conda-forge \
  openbabel \
  rdkit \
  && conda clean --all
```

---

Additionally we might want to add comments as to why we are installing some of these packages.

---

ok, sure

---

Is it possible to add a comment after the `\`  symbol? It didn't work for me.

---

Ah, didn't realize that. Then just in the commit message, because it will be visible via `git blame`.

---

ok